src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
```

Comment:
U can add extra file named normalise, and remove all there properties like that here.